src/app.py

```diff
@@ -68,30 +45,36 @@
 display_n_days_slider(st, p, DEFAULTS)
 
 # begin format data
-projection_admits = build_admissions_df(p.n_days, *p.dispositions)
-census_df = build_census_df(projection_admits, *p.lengths_of_stay)
+projection_admits = build_admissions_df(p=p) # p.n_days, *p.dispositions)
```

Comment:
projection_admits -> admissions_df

Reply:
what do you mean?

Reply:
Both the census and the admissions are projections, I'd just simplify the variable to admissions_df for consistency.

Reply:
i see. i agree with that. will resolve this conversation as soon as i push